docker-rebuild.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -ex
+# stop any running boulder containers
+docker-compose stop boulder
+# rebuild anything that has changed since the last run
+docker-compose build boulder
```

Comment:
Should this just be `docker-compose build` to future proof in case we introduce new buildable images?